### PR TITLE
Translation into English

### PR DESCRIPTION
**Pull request by tknrych**
Translated index.html and platforms.html into English.

1. index.html -> index_en.html
2. platforms.html -> platforms_en.html

### CODE REVIEW

**Comment by ytajima-git** [comment]
Dear @tknrych ,
Thank you for your interest in DoqueDB.
We have received your pull request, but unfortunately we are unable to 
accept your pull request, because we do not have a Contributor License
Agreement with you at this time.
Please follow the instructions in README.md  to create and send us a
personal or corporate Contributor License Agreement and send us your
pull request again.
Thank you for your cooperation.
                                                          Sincerely yours,
                                                          DoqueDB Development Team

**Comment by tknrych** [comment]
Hello @ytajima-git ,
I just submitted Contributor License Agreement to the designated email address. Please kindly review it and proceed your internal process. Thanks.

File: docs/index_en.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    DoqueDB is a RDBMS developed and published by Ricoh that features powerful Japanese full-text search.
+    This page will guide you through the various documents related to DoqueDB.
+  </p>
+
+  <dl>
+    <dt><a href="platforms_en.html">System Requirements</a></dt>
+    <dd>
+      You can check the system requirements for using DoqueDB.
+    </dd>
+    <dt><a href="howtouse_en.html">Let's use it.</a></dt>
```
Comment (htozaki) [inline docs/index_en.html:53]: In README.md, the name of this document is described as 'How to use'.

We would decide it later; this request would be merged as it is.

**Review by htozaki — APPROVED** [review]

File: docs/index_en.html (hunk 1 above)
Comment (tknrych) [reply docs/index_en.html:53]: Noted well. Thanks.

**Pull request merged**